.dccache

```diff
@@ -0,0 +1 @@
,1629723403370.2585,"b70a525e12362cfecf552f3ec2ccec4b1842a1425bb0a7facf716a6e0d86a1b4"],"/Users/ywei/Documents/repos/rasa/rasa/shared/core/training_data/story_writer/yaml_story_writer.py":[14146,1633360621607.3752,"06315f43764a2cd8886faf429d0e37ebae8102fad00fbd7b28927022f732b7dc"],"/Users/ywei/Documents/repos/rasa/rasa/shared/nlu/training_data/formats/__init__.py":[453,1633360621610.8867,"ad7dbcb13401b347cbe324ccb6fdf1565d83335e13bd69a49282c9b76908a21a"],"/Users/ywei/Documents/repos/rasa/rasa/shared/nlu/training_data/formats/dialogflow.py":[5944,1629723403374.4587,"a894b3016c9be6d94551139d5b8518433beb6e77ce7852c39c29a9ffbccb4369"],"/Users/ywei/Documents/repos/rasa/rasa/shared/nlu/training_data/formats/luis.py":[3020,1629723403374.6292,"7cc7e510c50d07f9b9af4bc2776cca0d46b74ff8b1d9cf948500407c36188b43"],"/Users/ywei/Documents/repos/rasa/rasa/shared/nlu/training_data/formats/rasa.py":[2799,1629723403375.1013,"fb7a2a858c59b51307f058d1034e4ef1f7304e703abaf16be02d50ae723c4027"],"/Users/ywei/Documents/repos/rasa/rasa/shared/nlu/training_data/formats/rasa_yaml.py":[22122,1633360621611.3083,"7885346d6959e1deb80a4cb7acb3eeed025caab9e3fd04b89581bbb47ec44817"],"/Users/ywei/Documents/repos/rasa/rasa/shared/nlu/training_data/formats/readerwriter.py":[5446,1633360621611.5933,"297863bb5f3b91c681187cb3bd888b0b73b9b7a77206c9ae420efbb33745b36e"],"/Users/ywei/Documents/repos/rasa/rasa/shared/nlu/training_data/formats/wit.py":[1820,1636468554780.8508,"3677da567cb7c8ef0139861965fa7beea977087760b2e8925765ad318970a7b9"],"/Users/ywei/Documents/repos/rasa/rasa/shared/nlu/training_data/schemas/__init__.py":[0,1629723403376.261,"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"],"/Users/ywei/Documents/repos/rasa/rasa/shared/nlu/training_data/schemas/data_schema.py":[2565,1633360621612.578,"fef9bbccec25ccb3e3761b2238a6f163b05f9213ea67b5a85b0058a726bd7793"],"/Users/ywei/Documents/repos/rasa/tests/shared/core/training_data/story_reader/__init__.py":[0,1629723403429.3376,"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"],"/Users/ywei/Documents/repos/rasa/tests/shared/core/training_data/story_reader/test_yaml_story_reader.py":[35894,1636995215045.8342,"4a0a409b419f65d2dd0abaee3fd2715b28829a4c0b86290e79519b698003978d"],"/Users/ywei/Documents/repos/rasa/tests/shared/core/training_data/story_writer/__init__.py":[0,1629723403429.976,"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"],"/Users/ywei/Documents/repos/rasa/tests/shared/core/training_data/story_writer/test_yaml_story_writer.py":[9414,1636966528547.1602,"5e8ae8eb952a04a0b82e93b54757c4eb8b674e04acf1111a0a8a3ec14ba9990e"],"/Users/ywei/Documents/repos/rasa/tests/shared/nlu/training_data/formats/__init__.py":[0,1629723403431.9187,"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"],"/Users/ywei/Documents/repos/rasa/tests/shared/nlu/training_data/formats/test_rasa_yaml.py":[13599,1633360621663.1018,"745619992c1e01211b8aaf8e301b23a7d2c235e723ef13d2127c1037606d38e9"]}
```

Comment:
Was this file supposed to be committed? Seems something that should go in `.gitignore` but I might be wrong 🤔

Reply:
Oh no. This is Snyk's cache file. Let me add it to `.gitignore`. 😬

Reply:
Force-pushed 😬 `.dccache` is gone.